src/storybook/stand-alone-documentaion/typography/text-styles/text-styles.jsx

```diff
@@ -74,9 +74,9 @@ export const TextStyles = () => {
       </VisualDescription>
       <VisualDescription
         ariaLabel="H5"
-        title="Fourth heading (Roboto 18px bold)"
+        title="Fourth heading (Roboto 16px bold)"
```

Comment:
(might be out of this PR's scope) looks like the details can easily get out of sync with the actual rules (meaning, the rules might change to be `Roboto 16px semi-bold`, but no one will remember to update this story).
Is there a way to automatically sync between the rules and the stories? perhaps [exporting a SASS variable](https://css-tricks.com/getting-javascript-to-talk-to-css-and-sass/#aa-sass-variables-and-javascript)?

Reply:
We exporting the css variables for this exactly. ׳e need to add them to this doc and i will talk with eivgany about the design of the doc with the variables.